### PR TITLE
fix: Profile fragment visible when logged out bug

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes : 192

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 1]
```diff
@@ -34,15 +35,18 @@ class ProfileFragment : Fragment() {
         startActivity(Intent(activity, MainActivity::class.java))
     }
 
+    override fun onAttach(context: Context?) {
+        super.onAttach(context)
+        if (!profileFragmentViewModel.isLoggedIn())
+            redirectToLogin()
+    }
```
[the remark this answers is not in the document]
Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: whats the error in indentation here? I use Android Studio's automatic indentation shortcut after making changes

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: Sorry, seemed like the curly brace was a tad bit left

**Comment by simarsingh24** [comment]
@iamareebjamal please merge this

File: app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json [hunk 2]
```diff
@@ -825,7 +825,7 @@
     ],
     "setupQueries": [
       "CREATE TABLE IF NOT EXISTS room_master_table (id INTEGER PRIMARY KEY,identity_hash TEXT)",
-      "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, \"a0ff61550af548e454d2446f54b579ee\")"
```
Comment (iamareebjamal) [inline app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json]: Why this change?

**Comment by simarsingh24** [comment]
resolved conflicts
